src/System.Private.Xml/src/System.Private.Xml.builds

```diff
@@ -3,12 +3,12 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Private.Xml.csproj" />
-    <Project Include="System.Private.Xml.csproj">
+    <!--<Project Include="System.Private.Xml.csproj">
```

Comment:
We do we not need these builds any longer? If we don't actually need them then we should remove them instead of commenting out. If we have to disable them temporarily then add a comment with a link to an issue to fix them.